test/e2e/getenvoy_extension_build_test.go

```diff
@@ -34,19 +34,17 @@ func TestGetEnvoyExtensionBuild(t *testing.T) {
 		test := test // pin! see https://github.com/kyoh86/scopelint for why
 
 		t.Run(test.String(), func(t *testing.T) {
+			// TODO: uses Docker, but may be possible to run parallel
```

Comment:
TODO, but not on this PR as we have more work to do. For example all the debug tests are e2e and should move here first